rotkehlchen/chain/substrate/utils.py

```diff
@@ -29,12 +29,25 @@ def is_valid_substrate_address(
 ) -> bool:
     """Validates a ss58 encoded substrate address for the given chain.
 
+    TODO: this function relies on py-scale-codec `ss58_decode()` for validating
```

Comment:
@LefterisJP I've added the comment regarding the matter

Reply:
thank you